src/schema/objects/columns.yaml

```
@@ -30,41 +34,48 @@ acq_time__sessions:
   format: datetime
 age:
   name: age
+  display_name: Subject age
```

do we document anywhere why/what 3 "things" are there to describe the same thing. It seems they are the 

```yaml
{id for a computer}:
  name: {something which is typically just an id but some times different... not sure why different at times, why cannot be the same as id}
  display_name: {a little more expanded for a human being but some times feels superficial ;) }
```

and should be 
```suggestion
  display_name: Participant age
```
to be more politically correct IMHO